dockers/docker-fpm-frr/docker_init.sh

```diff
@@ -86,13 +86,15 @@ elif [ "$CONFIG_TYPE" == "split-unified" ]; then
     rm -f /etc/frr/bgpd.conf /etc/frr/zebra.conf /etc/frr/staticd.conf
     write_default_zebra_config /etc/frr/frr.conf
 elif [ "$CONFIG_TYPE" == "unified" ]; then
+    echo "" >  /etc/frr/frr.conf
```

Comment:
Why do we need this? the below one should override the frr.conf file?

Reply:
To make sure it starts from  a clean file. During docker restart, the old frr.conf remains.

Reply:
The below code should override the frr.conf contents always, are we observing the stale content in the frr.conf during docker restart?
CFGGEN_PARAMS=" \
        -d \
        -y /etc/sonic/constants.yml \
        -t /usr/share/sonic/templates/gen_frr.conf.j2,/etc/frr/frr.conf

Reply:
Yes. I saw "fm add" got added twice in frr.conf.

Reply:
Add "fpm addr ..." to frr.conf.j2. 

Trigger "docker restart" and see proper output for "fpm addr ..."

Reply:
@venkatmahalingam this approach breaks sanity. 
json_path = 'tests/data/sonic-cfggen/frr.conf.j2/all.json'
match_path = 'tests/data/sonic-cfggen/frr.conf.j2/all.conf'

E         -  ['fpm address 127.0.0.1'],
ls ./src/sonic-bgpcfgd/tests/data/sonic-cfggen/frr.conf.j2/
all.conf        all.json

Reply:
Let's discuss in the Routing WG meeting 8/22